messages.js

```diff
@@ -158,9 +160,11 @@ exports.embedInventoryAsImage = async (event, locale) => {
   const description = l.__("KILL.VICTIM_INVENTORY", {
     victim: event.Victim.Name,
   });
-  const footer = good
-    ? l.__("KILL.FOOTER_GOOD", { killer: event.Killer.Name })
-    : l.__("KILL.FOOTER_BAD", { killer: event.Victim.Name });
+  const footer = {
+    text: good
+      ? l.__("KILL.FOOTER_GOOD", { killer: event.Killer.Name })
+      : l.__("KILL.FOOTER_BAD", { killer: event.Victim.Name }),
+  };
```

Comment:
This can be a function since it's the same implementation as in `embedEventAsImage`.